src/app/project-report/project-report.page.html

```diff
@@ -90,4 +90,17 @@ <h1 class="modal-heading"><strong>No Data available</strong></h1 >
       </ion-content>
     </ng-template>
   </ion-modal>
-</ion-content>
\ No newline at end of file
+  <ion-modal [isOpen]="OpenForCopyLink" class="copy-link-popup" (ionModalDidDismiss)="onModalDismiss()">
```

Comment:
Use camel case naming convention for "OpenForCopyLink"

Reply:
Can this Popup be made as a generic component so that it can be triggered from a service or ts file

Reply:
done